src/entry.hpp

```diff
@@ -389,17 +390,20 @@ class Entry : public node::ObjectWrap
     {
         if (status < 0)
         {
+            std::cout << "MAP ERROR INTERNAL" << std::endl;
```

Comment:
I guess that's temporary...

Reply:
Yes, I'm still debugging NodeJS (and C#)